version.php

```diff
@@ -24,11 +24,12 @@
 
 defined('MOODLE_INTERNAL') || die();
 
-$plugin->version = 2020101200;
-$plugin->release   = 2020101200;      // Same as version.
+$plugin->version = 2022011600;
```

Comment:
this one should be a micro bump

Reply:
Sorry Brendan, this has been fixed, Please review.. Thank you...